user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_diagnostic_procedures/SLNetClientTest_inspecting_active_replication_buffers.md

```diff
@@ -26,8 +26,46 @@ To drop one specific replication buffer:
 
 1. In the *Type* field at the bottom, select *ReplicationBufferStats*.
 
-> [!NOTE]
-> When a replication buffer saves files to disk, those files are located in C:\\Skyline DataMiner\\System Cache\\SLNet and are named "ReplicationBuffer_clienthostname_ip_dmaid_eid_objectid.bin".
-
 > [!WARNING]
 > Always be extremely careful when using the SLNetClientTest tool, as it can have far-reaching consequences on the functionality of your DataMiner System.
+
+## Managing replication buffer files
+
+When a replication buffer saves files to disk, those files are located in `C:\Skyline DataMiner\System Cache\SLNet` and are named as follows:
+
+- From DataMiner 10.3.0 [CU16]/10.4.0 [CU4]/10.4.7 onwards<!--RN 39428-->: `ReplicationBuffer_clienthostname_ip_dmaid_eid.bin`
+
+  These files do not include a unique hash in their filenames. As a result there is only one file per replicated element.
```

Comment:
@mmphs, I have restructured the documentation so that both an example of the new and old filenames is provided.

Reply:
Looks great!